swig/sundials/fsundials.i

```diff
@@ -57,6 +57,9 @@
 // Assume sunrealtype* is an array of doubles
 %apply double[] { sunrealtype* };
 
+// Assume realtype* is an array of doubles
+%apply double[] { sunrealtype* };
+
```

Comment:
This duplicates the above lines.  I'm assuming that this should have been
```suggestion
// Assume realtype* is an array of doubles
%apply double[] { realtype* };

```
but now that `realtype` is deprecated, maybe it should now be removed entirely:
```suggestion
```